solutions/secure/variables.tf

```diff
@@ -116,12 +116,12 @@ variable "kms_endpoint_type" {
 
 variable "existing_kms_key_crn" {
   type        = string
-  description = "The existing root key CRN of the Hyper Protect Crypto Service (HPCS) to use for disk encryption."
+  description = "The CRN of an existing Hyper Protect or Key Protect root key to use for disk encryption."
```

Comment:
Can you update this to mention that is none is specified, one will be created in the KMS instance provided in the `existing_kms_instance_guid` input.